main.c

```diff
@@ -82,6 +82,13 @@
 #define  LED3_ON 0x06
 #define  LED3_OFF 0x07
 
+#define UARTMUXSEL_SET_MIKROBUS 0x08
+#define UARTMUXSEL_SET_MSP430 0x09
+#define UARTMUXSEL_GET 0x0C
+
+#define UARTMUXSEL_MODE_MIKROBUS 0x0A
+#define UARTMUXSEL_MODE_MSP430   0x0B
+
```

Comment:
It is slightly confusing that the commands are different than the mode bits and that the mode bits aren't simply 0/1, but I think this is OK.